### DISCUSSION TITLE
Change hasura cloud region

### DISCUSSION THREAD

**Discussion by ghost** [hasura/graphql-engine#8587] [Question] [answered]
On Hasura Cloud, can I change my instance Region after it is created?

**Comment by akhil-naidu** [comment, marked as answer]
> Yes, you can change your region

Currently, the Region change is only available for Standard tier projects. If you have a Standard tier project, changing the region takes minutes.

You can refer to our [official documentation](https://hasura.io/docs/latest/graphql/cloud/projects/regions/#changing-region-of-an-existing-project) for more information on changing your region.

**Quick Note**: There might be a short downtime while your project is being moved to a different region. If your database allows connections only from specific IP addresses, make sure you add the Hasura Cloud IP of this new region to the list of allowed IP addresses.

There is a temporary limitation on the older version: If your project was considerably senior, i.e., you are using Hasura v2.0 and below, then you need to follow specific manual instructions.

> Also, stay tuned for multi-region support.

**Reply by meetzaveri** [reply]
This looks good to me. Also, to note that if you are dealing with any API that Hasura cloud exposes, you'll can pass  value to `region` field(in any case its optional). An example would be of to configure github action file for preview apps creation (which uses API exposed by Hasura cloud for creating cloud apps for preview purposes).
